### DISCUSSION TITLE
Disabling "Remember Me" Function via docker-compose.yml doesn´t work

### DISCUSSION THREAD

**Discussion by bokkabonga** [dani-garcia/vaultwarden#1338] [Q&A]
### Subject of the issue
After disabling the DISABLE_2FA_REMEMBER via the docker-compose.yml the option still works.

### Your environment

* Bitwarden_rs version: 1.18

* Install method: Built from Source
* Reverse proxy and version: Apache2.4.6
* Version of mysql/postgresql: mariadb 10.4
* Other relevant information:

### Steps to reproduce
Set DISABLE_2FA_REMEMBER: 'true' in the docker-compose.yml. I am aware, that the syntax is slightly different from the [template](https://github.com/dani-garcia/bitwarden_rs/blob/master/.env.template) but it works fine with all other settings, so i assume it should work here.

Restartet bitwarden after changing the docker-compose.

### Expected behaviour
When checking the "remember me" box, it should not work anymore. The checkbox on the admin-page should be checked.

### Actual behaviour
The Option is still working and the checkbox ist not checked. Other changes to the docker-compose were applied however.

**Comment by BlackDex** [comment]
You maybe have a settings.json in your data folder. If that is the case some settings could be overridden by this. This is also visible right at the start of the container `docker logs bitwarden_rs`
